tests/unit/io/WKTReaderTest.cpp

```diff
@@ -473,4 +486,35 @@ void object::test<24>
     ensure_equals(geom->getNumGeometries(), 3u);
 }
 
+// Raise exception on strict mode
+template<>
+template<>
+void object::test<25>
+()
+{
+    struct test_wktreader_strict_mode mode(this);
+    // codespell:ignore POIN
```

Comment:
Maybe just put this in ./tools/codespell.ignore

Reply:
Or just pick a shorter typo that doesn't trigger codespell like `POI(1 1)`. (The other typos below never applied, it only took exception to POIN)